### PR TITLE
Fixed telnet Docs

### PR DESCRIPTION
**Pull request by Yashvendra**
Fixed #11439 `auxiliary/scanner/telnet/telnet_login` and `auxiliary/scanner/telnet/telnet_version` modules documentation.

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/telnet/telnet_login.md [hunk 1]
```diff
@@ -7,12 +7,11 @@ This module will test a telnet login with a list of provided credentials on a ra
 2. Do: ```set RHOSTS [IP]```
 3. Do: ```set THREADS [NUMBER OF THREADS]```
 4. Do: ```set USER_FILE [USERNAME FILE]```
-5. Do: ```set PASS_FILE[PASSWORD FILE]```
+5. Do: ```set PASS_FILE [PASSWORD FILE]```
 6. Do: ```run```
 
 ## Options
 
-This auxiliary module allows you to pass credentials in a number of ways. You can specifically set a username and password, you can pass a list of usernames and a list of passwords for it to iterate through, or you can provide a file that contains usernames and passwords separated by a space.
```
Comment (bcoles) [inline documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: Is there a reason this was removed?

Comment (Yashvendra) [reply documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: @h00die pointed out that it is more or less the same as description.

Comment (h00die) [reply documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: I would move it up to the description

Comment (Yashvendra) [reply documentation/modules/auxiliary/scanner/telnet/telnet_login.md]: Done.

**Comment by h00die** [comment]
# Release Notes
This PR adds improvements to the telnet scanner docs

**Comment by h00die** [comment]
Thanks for the patience @Yashvendra while we worked through all this!

**Pull request merged**